src/services/databaseConnector.ts

```diff
@@ -0,0 +1,8 @@
+import mysql from "promise-mysql";
+import { dbSettings } from "../../config/dbConfig";
```

Comment:
Can you include an example config file in the location it should be, clearly marked as config.example.js? ts? or whatever? So that people can download your project and figure out how to use it.

For a config file I would recommend either a js/ts file in the root that exports needed stuff, rather than something nested in your folder structure, just so that it's easier for people downloading your project (e.g. me) to find and adjust to their needs.

A single config file should be sufficient, you can always export the different config kinds as separate things if you want to. I expect database details will be the only config required for this project anyway.